src/mobility/components/CarImage.tsx

```diff
@@ -1,35 +1,63 @@
-import {Image, View} from 'react-native';
+import {Image, ImageStyle, View} from 'react-native';
 import React from 'react';
 import {ThemeIcon} from '@atb/components/theme-icon';
-import {Car} from '@atb/assets/svg/mono-icons/transportation';
+import {Car} from '@atb/assets/svg/mono-icons/transportation-entur';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {ThemeText} from '@atb/components/text';
 
 type Props = {
-  uri: string | undefined;
+  uri?: string | undefined;
+  plus?: number | undefined;
 };
-export const CarImage = ({uri}: Props) => {
+
+export const CarImage = ({uri, plus}: Props) => {
+  const style = useSheetStyle();
+
   return uri ? (
     <Image
-      style={{height: 75, width: 100, resizeMode: 'contain'}}
+      style={[style.container as ImageStyle, style.image as ImageStyle]}
```

Comment:
I haven't tried, but I suppose this could be solved by creating a separate image style with a type to `createThemeHook`? I.e. something like this: `const useImageStyles = StyleSheet.createThemeHook<ImageStyle>(...)`

Reply:
I tried but this doesn't work

```
const useImageStyle = StyleSheet.createThemeHook<ImageStyle>(() => {
  return {
    image: {
      resizeMode: 'cover',
    } as ImageStyle,
    defaultImage: {
      justifyContent: 'center',
      alignItems: 'center',
    } as ImageStyle,
  };
});
```

Here's the error
```
Argument of type '() => { image: ImageStyle; defaultImage: ImageStyle; }' is not assignable to parameter of type 'ThemedStyles<NamedStyles<ImageStyle>>'.
  Type '{ image: ImageStyle; defaultImage: ImageStyle; }' has no properties in common with type 'NamedStyles<ImageStyle>'.ts(2345)
```

Reply:
Ok, I see. It's not a big deal. 

I know you haven't worked with Typescript before and it's important to note that adding  `as ...` is not a type cast in a traditional sense, but rather a way of forcing Typescript into accepting whatever as your desired type. You circumvent the type system in a way and as such `as ...` should be used as sparingly as possible.

Reply:
Thanks for the explanation, I'll try to look for other solution so we can limit the usage of `as ...`